apps/token/src/routes/governance/components/propose/proposal-form-vote-and-enactment-deadline.tsx

```diff
@@ -1,29 +1,35 @@
 import { useEffect, useState, useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
-import { parse as ISO8601Parse, toSeconds } from 'iso8601-duration';
 import {
   ButtonLink,
   FormGroup,
   Input,
   InputError,
 } from '@vegaprotocol/ui-toolkit';
 import { addHours, addMinutes } from 'date-fns';
+import {
+  deadlineToSeconds,
+  secondsToRoundedHours,
+} from '@vegaprotocol/governance';
 import { ProposalFormSubheader } from './proposal-form-subheader';
 import type { UseFormRegisterReturn } from 'react-hook-form';
 
 interface DeadlineProps {
   vote: number;
-  enactment: number;
+  enactment: number | undefined;
   validation: number;
 }
 
 interface DeadlineDatesProps {
   vote: Date;
-  enactment: Date;
+  enactment: Date | undefined;
   validation: Date;
 }
 
 interface ValidationFormProps {
+  onValidationMinMax:
```

Comment:
chore (blocking): optional butI think this is always passed in? Or am I missing something